ts/context.ts

```diff
@@ -378,9 +378,9 @@ export class QuickJSContext implements LowLevelJavascriptVm<QuickJSHandle>, Disp
    */
   newArrayBuffer(buffer: ArrayBufferLike): QuickJSHandle {
     const array = new Uint8Array(buffer);
-    const ptr = this.memory
+    const handle = this.memory
       .newHeapBufferPointer(array)
-      .consume((bufferHandle) => this.ffi.QTS_NewArrayBuffer(this.ctx.value, bufferHandle.value.pointer, array.length))
```

Comment:
removing this line seemed to fix the issue. Not sure if it'll cause more issues, but it seems to be fine